product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_sql_tutorial/01_sample_database/01_sample_database_installation.mdx

```diff
@@ -20,4 +20,4 @@ The tables and programs are created in the first schema of the search path in wh
 SHOW SEARCH_PATH;
 ```
 
-You can alter the search path using commands in PSQL.
+You can alter the search path using commands in PSQL (a terminal-based front-end to PostgreSQL).
```

Comment:
PSQL is used in both PostgreSQL and EPAS. I think it should be Postgres here to generalize
```suggestion
You can alter the search path using commands in PSQL (a terminal-based front-end to Postgres).
```

Reply:
Got it, thanks, @nidhibhammar ! Fixing now, along with some other tweaks.